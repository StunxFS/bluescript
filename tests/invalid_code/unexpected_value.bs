enum MyEnum {
    a
}

fn main() {
    const x = MyEnum;
}