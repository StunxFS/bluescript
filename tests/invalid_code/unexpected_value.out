tests/invalid_code/unexpected_value.bs:6:15: error: expected value, found enum `MyEnum`
